prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.ts

```diff
@@ -108,7 +112,8 @@ export class EnrolleeReviewComponent {
       : [];
   }
 
-  public onRoute(routePath: string): void {
+  public onRoute(routePath: string | (string | number)[], event: Event = null): void {
+    event?.preventDefault();
```

Comment:
Did you find the event wasn't available?  I would have thought the event would always be present since it bubbles up from the element.  So I would suspected it doesn't need a default value or a null check.